wordpress-mobile/gutenberg-mobile
# Feat: support of multiple selection in media upload

In this PR I added support for multiple selections of media in MediaUpload component.  It is a breaking change and apps will crash without changes from listed PRs.

Associated issue - https://github.com/wordpress-mobile/gutenberg-mobile/issues/1298

Related gutenberg PR - https://github.com/WordPress/gutenberg/pull/17397
Related Wordpres-iOS PR - https://github.com/wordpress-mobile/WordPress-iOS/pull/12404
Related Wordpress-Android PR - https://github.com/wordpress-mobile/WordPress-Android/pull/10556
To test:
- set multiple prop to true in image/video block
- run Wordpress-iOS or Wordpress-Android
- add image/video block - multiple select should be enabled for selecting media from the device or WP Media
- the array of selected media should be passed to the callback function

NOTE: Multiple selections of media from the device on iOS doesn't work since this functionality is implemented by the client app. Our example app from `ios` directory doesn't support multiple selections from the device because it uses build-in `UIImagePickerController`. In order to make it works in our example app, I would need to add an external library that supports multiple selections.
